### PR TITLE
method for urls endpoint

### PR DESCRIPTION
**Pull request by khan08**

### CODE REVIEW

File: indexclient/client.py [hunk 1]
```diff
@@ -295,6 +295,31 @@ def list_versions(self, did):
...
+        params = {
+            "url": url,
+            "key": key,
+            "value": value,
+            "fields": fields,
+            "version": version,
+            "limit": limit,
+            "offset": offset
+        }
+        resp = self._get("index/urls/metadata/q", params=params)
```
Comment (kulgan) [inline indexclient/client.py]: path was updated to `_query/urls/metadata`

File: indexclient/client.py [hunk 2]
```diff
@@ -295,6 +295,31 @@ def list_versions(self, did):
...
+    def query_url(self, exclude=None, include=None, version=False, fields=None, limit=100, offset="0"):
+        params = {
+            "exclude": exclude,
+            "include": include,
+            "version": version,
+            "fields": fields,
+            "limit": limit,
+            "offset": offset
+        }
+        resp = self._get("index/urls/q", params=params)
```
Comment (kulgan) [inline indexclient/client.py]: update path `_query/urls/q`

File: indexclient/client.py [hunk 3]
```diff
@@ -295,6 +295,31 @@ def list_versions(self, did):
             versions.append(Document(self, version["did"], version))
         return versions
 
+    def query_urls_metadata(self, url, key, value, fields=None, version=False, limit=100, offset="0"):
+        params = {
+            "url": url,
+            "key": key,
+            "value": value,
+            "fields": fields,
+            "version": version,
```
Comment (kulgan) [inline indexclient/client.py]: version is now `versioned`

File: indexclient/client.py [hunk 4]
```diff
@@ -295,6 +295,31 @@ def list_versions(self, did):
...
+            "offset": offset
+        }
+        resp = self._get("index/urls/metadata/q", params=params)
+        return resp.json()
+
+    def query_url(self, exclude=None, include=None, version=False, fields=None, limit=100, offset="0"):
+        params = {
+            "exclude": exclude,
+            "include": include,
+            "version": version,
```
Comment (kulgan) [inline indexclient/client.py]: version is now `versioned`

File: indexclient/client.py [hunk 5]
```diff
@@ -295,6 +295,31 @@ def list_versions(self, did):
...
+            "value": value,
+            "fields": fields,
+            "versioned": versioned,
+            "limit": limit,
+            "offset": offset
+        }
+        resp = self._get("_query/urls/metadata/q", params=params)
+        return resp.json()
+
+    def query_url(self, exclude=None, include=None, versioned=False, fields=None, limit=100, offset="0"):
```
Comment (jbarno) [inline indexclient/client.py]: nitpick: fields and versioned are switched between these signatures

**Pull request closed without merging**